src/custom-elements-es5-adapter.ts

```diff
@@ -103,62 +82,15 @@ export function addCustomElementsEs5Adapter(html: string): string {
-  const bodyElement = dom5.query(parsed, bodyMatcher)!;
   const loaderScriptUrl = dom5.getAttribute(script, 'src')!;
   const adapterScriptUrl =
       url.resolve(loaderScriptUrl, 'custom-elements-es5-adapter.js');
-  const es5AdapterScript = parse5.parseFragment(`
-    <!-- This block adds the "custom-elements-es5-adapter.js" shim to your application.
-         It includes special logic that removes the shim before it can be parsed or run
-         on browsers without native support for ES2015+ & custom elements. -->
-    <div id="autogenerated-ce-es5-shim">
-      <script type="text/javascript">
-        if (!window.customElements) {
-          var ceShimContainer = document.querySelector('#autogenerated-ce-es5-shim');
-          ceShimContainer.parentElement.removeChild(ceShimContainer);
-        }
-      </script>
-      <script type="text/javascript" src="${adapterScriptUrl}"></script>
-    </div>
-  `);
-  if (!bodyElement.childNodes) {
-    bodyElement.childNodes = [];
-  }
-
-  // If script is already in the body, just insert the shim right before it.
-  if (dom5.nodeWalkAncestors(script, bodyMatcher)) {
-    dom5.insertBefore(bodyElement, script, es5AdapterScript);
-
-  } else {
-    // Otherwise we need to move the webcomponents-loader/webcomponents-lite
-    // loader down to the body so that the es5 adaperter script shim will work.
-    // Any following scripts, styles, and imports need to be moved down to the
-    // body as well to preserve ordering.
-    const orderSensitiveSiblings = dom5.queryAll(
-        script.parentNode!, p.OR(scriptMatcher, styleMatcher, linkMatcher));
-    const scriptIndex = orderSensitiveSiblings.indexOf(script);
-    const afterScriptSiblings = orderSensitiveSiblings.slice(scriptIndex);
-    const loaderMovedComment = dom5.constructors.comment(
-        ' AUTO-GENERATED BY polymer-cli: "webcomponents-loader.js" and ' +
-        'related resources have been moved to <body> so that ' +
-        '"custom-elements-es5-adapter.js" polyfill could be properly ' +
-        'injected into this file. ');
-
-    // Insert a "loader moved" comment fragment into the head.
-    dom5.insertBefore(headElement, script, loaderMovedComment);
-    // Move elements to the top of the body in reverse order, so that the first
-    // moved element is prepended last, preserving the original order.
-    afterScriptSiblings.reverse();
-    for (const releventScript of afterScriptSiblings) {
-      dom5.insertBefore(
-          bodyElement, bodyElement.childNodes[0] || null, releventScript);
-    }
-    // Finally, inject the es5-adapter shim at the top of the body.
-    dom5.insertBefore(
-        bodyElement, bodyElement.childNodes[0] || null, es5AdapterScript);
-  }
+  const es5AdapterFragment = parse5.parseFragment(`
+    <script>if (!window.customElements) { document.write('<!--'); }</script>
+    <script type="text/javascript" src="${adapterScriptUrl}"></script>
+    <!-- do not remove -->
```

Comment:
@azakus what was the incantation to prevent the html minifier from removing this? Was it `<!-- !do not remove -->`